std_msgs/Empty goal
---
geometry_msgs/Point[] list_of_odoms
---
float32 current_total